GNUmakefile

```diff
@@ -17,10 +17,25 @@ GOPATH=$(shell go env GOPATH)
 MAIN_GOPATH=$(shell go env GOPATH | cut -d: -f1)
 
 ASSETFS_PATH?=agent/uiserver/bindata_assetfs.go
-# Get the git commit
+
+# If running inside a git repository, store repo status info in environment variables
+IS_INSIDE_GIT_REPO="$(git rev-parse --is-inside-work-tree 2>/dev/null)"
+GIT_DIRTY_DEFAULT=true
```

Comment:
What should the value of `GIT_DIRTY` be if the repo is pristine? It looks like the code meant to set it to `true`, but I think it's not using the test command correctly... shouldn't it be `|| echo true`, not `|| true`?

If I'm correct, I'd need to fix my next version of this line as well to include the `echo` after the `||`.

 What should the value of GIT_DIRTY be if building inside a docker container with no access to the .git folder? The value I currently have probably isn't right.